### ISSUE TITLE
Ability to "centre" a servo

### ISSUE THREAD

**Issue by swinguk** [david284/MMC-SERVER#53]
Whilst working with some servos in dingo mounts this weeks, at many points during the fitting / testing process, you really need to centre the servo to aid the fitment. It would be useful for a button / option to be able to "centre" a servo (in principle the value halfway between the min and max values, ie in the case of CANMIOs, I think that's 128). Whilst it's possible to set either the ON or OFF position to the centre value, there's value in my opinion for being able to centre a servo without updating the on/off values.

**Comment by david284** [comment]
Mocked up a 'center' option - but only if 'outputOnWrite' set, so doesn't appear for all sliders
I think it would be good for all 'linear' outputs, but I suspect that only 'servo' type outputs set the output value when the NV is written, so I think it only works for servo outputs

If you think that's ok, I'll look to include it in the next set of changes

![image](https://github.com/user-attachments/assets/a7e31044-b0d9-4959-aaf2-96a7194b06d2)

**Comment by swinguk** [comment]
Yes, that looks good to me. Whilst I appreciate there may be value in adding it to other output types, it's only the servo where I have a use case for needing to centre the output, due to the use of mounts that benefit from having the servo in the centre for fitment.

**Comment by david284** [comment]
added in b163edbed65dc1644cc38900d8c1f4fb534babea
